### ISSUE TITLE
Bug: missing hyprland's `special` workspace

### ISSUE THREAD

**Issue by ozwaldorf** [Alexays/Waybar#1983]
Using `hyprctl workspaces`, it lists workspace `-99`, named `special`. Waybar doesn't pick this up, but it'd be nice to be able to add it without a custom module

**Comment by lllldronellll** [comment]
I agree, I think many users would really appreciate it's implementation, as Hyprland's approach to minimizing windows lies in it's special workspaces feature. 

Thanks so much!

**Comment by khaneliman** [comment]
> Using hyprctl workspaces, it lists workspace -99, named special. Waybar doesn't pick this up, but it'd be nice to be able to add it without a custom module

Is this resolved for you with using the `show-special` ?

**Comment by Alexays** [comment]
Closing — this is fixed/implemented on current master and will be in the next release. Please reopen if it still occurs after updating. 🙏
